src/main/java/reactor/kafka/receiver/ImmutableReceiverOptions.java

```diff
@@ -30,13 +38,6 @@
 import java.util.regex.Pattern;
 import java.util.stream.Collectors;
 
-import org.apache.kafka.clients.consumer.ConsumerConfig;
-import org.apache.kafka.common.TopicPartition;
-import org.apache.kafka.common.config.ConfigException;
-import org.apache.kafka.common.serialization.Deserializer;
-import reactor.core.scheduler.Scheduler;
-import reactor.core.scheduler.Schedulers;
```

Comment:
Is it correct for this project to rearrange imports?

Not related, but in Spring Integration 6.0 I'm going to follow IDE settings from Spring Framework: https://github.com/spring-projects/spring-integration/issues/3686
From where I believe the import order is exactly like it was before your refactoring here.

Reply:
We had nothing like that when this syntax used to work 

    compile ("com.fasterxml.jackson.core:jackson-core:$jacksonVersion", optional)

https://github.com/spring-projects/spring-kafka/blob/5c54ad02339ab1849f98fd3e785f811c6deb17c5/build.gradle#L198

Reply:
Yes. I don't mind, but turns out Gradle has decided to go further and above limited number of Maven dependency variants.
But unfortunately they liquidated an `optional` as out-of-the-box at all...

Reply:
in core we do use `compileOnly` (but indeed these don't appear in the pom, see https://github.com/reactor/reactor-core/issues/2262)